spec/manager.spec.js

```diff
@@ -0,0 +1,59 @@
+const Manager = require("../src/manager.js");
+const Cohort = require("../src/cohort.js");
+const Student = require("../src/student.js");
+
+describe("Manager", () => {
+  it("create a cohort name", () => {
```

Comment:
<img width="445" alt="Screenshot 2022-04-04 at 10 06 48" src="https://user-images.githubusercontent.com/98287394/161511496-096bb94a-e126-40fd-afec-5810727d90d3.png">
You have a test for creating a cohort name but do you have a test for creating the cohort itself? I would create a new cohort with:
 `const expected = [ 'Cohort x' ] // setup`
 `const result = cohortManager.createCohort( 'Cohort x' ) // execute`
 `expect(result).toEqual(expected // verify`
And then create the cohort itself